reference/5.1/Microsoft.PowerShell.Utility/Write-Output.md

```diff
@@ -28,10 +28,12 @@ If the command is the last command in the pipeline, the object is displayed in t
 `Write-Output` sends objects down the primary pipeline, also known as the "output stream" or the
 "success pipeline." To send error objects down the error pipeline, use Write-Error.
 
-This cmdlet is typically used in scripts to display strings and other objects on the console.
-However, because the default behavior is to display the objects at the end of a pipeline, it is
-generally not necessary to use the cmdlet. For instance, `Get-Process | Write-Output` is equivalent
-to `Get-Process`.
+> [!NOTE] This cmdlet is typically used in scripts to display strings and other objects on the
+> console. One of the built-in aliases for `Write-Output` is `echo` and similar to other shells that
+> use `echo`, the default behavior is to display the objects at the end of a pipeline. In
+> PowerShell, it is generally not necessary to use the cmdlet in instances where the output is
+> displayed by default. For example, `Get-Process | Write-Output` is equivalent to `Get-Process`.
+> Or, `echo "Home directory: $HOME"` can be written, `"Home directory: $HOME"`.
```

Comment:
I like it, but I still think that this is the place to _briefly_ note the one use where explicit use of `Write-Output` _is_ useful: `Write-Output -NoEnumerate`, to output a collection _as a single object_, so that the next downstream command receives it as such.

You could just point to the `-NoEumerate` parameter description for details. Incidentally, I think the current description doesn't do a great job of clarifying the parameter's purpose, and "The NoEnumerate parameter has no effect on collections that were created by wrapping commands in parentheses" is downright confusing. What matters is that if you wrap the `Write-Output` command _as a whole_ in `(...)`, the effects of `-NoEnumerate` are negated.

Also - sorry to pack things on here - I don't think the examples should use `@(...)` so as to perpetuate the misconception that `@(...)` is needed for array literals.

Reply:
No need to be sorry, this is all great input. 

1. To the first point, I went ahead and added it to the descriptiont. However, I took out the NOTE identifier as I think adding all of this info will read better as just a full description. Let me know what you think about the explaination and if you have a reason for keeping it in a NOTE format. 
2. I agree with you here and I went ahead and updated to show the actual behavior. I think it's clear but feel free to let me know what you think.
3. Not at all. This is true and I struggle with using the operator unnecessarily. I've removed them from the examples

Reply:
All looks good now - thank you.